torchtext/data/functional.py

```diff
@@ -34,7 +33,11 @@ def generate_sp_model(filename, vocab_size=20000,
         >>> from torchtext.data.functional import generate_sp_model
         >>> generate_sp_model('test.csv', vocab_size=23456, model_prefix='spm_user')
     """
-
+    try:
+        import sentencepiece as spm
+    except ImportError:
+        print("Please install sentencepiece")
```

Comment:
It's better to use ["ImportWarning"](https://docs.python.org/3.8/library/exceptions.html#ImportWarning) here